nav2_util/package.xml

```diff
@@ -10,6 +10,7 @@
   <license>BSD-3-Clause</license>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
+  <build_depend>libboost-program-options-dev</build_depend>
```

Comment:
Can we add this as a `<depend>` like the others in this file?

Reply:
In this case, there is a difference between the build depend and the exec depend. I don't want someone installing the binary package to get the boost-dev package as a dependency. That would bring in all the header files and stuff that the user doesn't need. They should just get the library (non-dev) package

I believe making the distinction between build and exec depend is the right thing to do for this dependency

Reply:
Oh sorry, I missed that they were different packages. You're correct